src/resolve/resolved-resolv-conf.c

```diff
@@ -328,6 +328,12 @@ static int write_stub_resolv_conf_contents(FILE *f, OrderedSet *dns, OrderedSet
         return fflush_and_check(f);
 }
 
+void manager_write_non_stub_resolv_conf(Manager *m) {
```

Comment:
please call this "manager_symlink_stub_to_uplink_resolv_conf()" please, because this doesn't "write" anything, but it just redirects the stub resolv.conf to the uplink one